### PR TITLE
capabilities: update documentation

### PR DESCRIPTION
**Pull request by alban**
Follow-up to https://github.com/kinvolk/inspektor-gadget/pull/1035

While rebuilding the gif videos, I encountered some problems, see individual commits.

See also #1038.

### CODE REVIEW

File: tools/demos/Makefile [hunk 1]
```diff
@@ -23,6 +23,11 @@ all-gadgets: $(addsuffix /demo.gif,$(DEMO_LIST_EXCEPT_INSTALL))
 	agg $*/demo.cast $*/demo.gif
 	ls -lh $*/demo.gif
 
+.PHONY: check-terminal
+check-terminal:
+	test `tput lines` -eq 24
```
Comment (mauriciovasquezbernal) [inline tools/demos/Makefile:28]: What about using https://stackoverflow.com/a/8308063 to try to set the size before testing it?

**Comment by mauriciovasquezbernal** [comment]
Given that you're updating all the gifs. What do you think about using the new "kubectl gadget version" command (https://github.com/kinvolk/inspektor-gadget/pull/977) instead of https://github.com/kinvolk/inspektor-gadget/blob/main/tools/demos/install/demo.sh#L25?

**Comment by alban** [comment]
Updated

**Comment by blanquicet** [comment]
> It seems to show the versions of the client and the server in a different way:
> 
> ```
> $ kubectl gadget version
> Client version: v0.9.1-47-g269ceca6-dirty
> Server version: alban_caps_prefix2
> ```
> 
> Should I use it anyway?

It will have to be update it to use the next released CLI and image. In that way, both will be tagged and the output would be:

$ kubectl gadget version
Client version: v0.10.0
Server version: v0.10.0

**Review by blanquicet — APPROVED** [review]
LGTM! But I think having the capabilities column with a fixed size is not the best solution. It is taking up too much space without the need.

**Comment by alban** [comment]
> LGTM! But I think having the capabilities column with a fixed size is not the best solution. It is taking up too much space without the need.

Right, in most cases it takes too much space because the capability with the long name is not the one frequently used. But it is the most important column though... Any suggestions?

**Pull request merged**